documentation-site/examples/accordion/renderpanelcontent.js

```diff
@@ -6,7 +6,7 @@ const content =
   'Praesent condimentum ante ac ipsum aliquam, ac scelerisque velit sagittis. Ut sit amet libero scelerisque, accumsan ante vitae, hendrerit tellus. Nullam metus est, vehicula a aliquet id, lobortis in mauris.';
 
 export default () => (
-  <Accordion renderPanelContent>
+  <Accordion renderAll>
```

Comment:
it's not a breaking change, but now it supports both, to have the same API as the Tab comonent